docs/blog/20230904-Daeraxa-SeptemberUpdate.md

```diff
@@ -0,0 +1,89 @@
+
+And what it looks like if you turn the setting off to remove the tab from the title:
+
+<img src="./assets/title-bar-no-tab.png" height="50" />
+
+This is currently available in our [rolling release](https://pulsar-edit.dev/download.html#rolling-release) and will be in our next regular release in a couple of weeks.
+
+## Unpublishing packages
+
+<img src="./assets/package.png" height=200>
+
+There are some very good reasons to want to unpublish a package or a specific version from the Pulsar Package Repository. You may have made a mistake or simply want to deprecate an old, non-functional version. Either way, we had an issue; it just wasn't working. Thankfully, due to some great collaboration between [@asiloisad/@bacadra] and [@confused-techie], the problem was [found and patched](https://github.com/pulsar-edit/package-backend/pull/195).
+
+However, shortly after this was fixed, we were contacted by a community member asking why they were unable to re-publish their package after a successful unpublish in order to fix a versioning mistake. Ultimately, we realised that the wording shown during the unpublishing process does not make it clear that once a package has been unpublished, we *permanently block the package name from being used ever again*. This is intentional in order to prevent any bad actors from hijacking the name of a previous package in order to hide malicious code. This also provided the opportunity for us to come up with a process to work around this in exceptional circumstances, we will now ensure that:
+
+1. Nobody has downloaded the unpublished package.
+2. There are no significant code changes between the unpublished version and the version the author wishes to re-publish.
+3. The author can prove ownership of the repository.
+
+So with the above process and an improvement to the wording in the `unpublish` command coming soon, we hope that this situation can be avoided in the future. A full writeup of what was done has been added to our [admin actions log](https://github.com/pulsar-edit/package-backend/pull/198) that details events just like this for full transparency to the community.
+
+## Pulsar cleanup
+
+<img src="./assets/cleaning.png" height=200>
+
+It was brought to our attention by a community member that, upon uninstall, Pulsar was not clearing up all the directories it created during installation. Whilst this is somewhat expected for the configuration directory, there were also examples of other elements being left behind. To this end, [@confused-techie] has been putting together a new package, [pulsar-cleanup](https://github.com/pulsar-edit/pulsar-cleanup), to try and deal with these leftover elements. As `electron-builder` is largely in control of the uninstall process, some elements do get left over at the end of the process. This utility will be a stand-alone executable that can be used to fully clean up all these extra artefacts, leaving your system entirely clean of any Pulsar installation.
```

Comment:
```suggestion
It was brought to our attention by a community member that, upon uninstall, Pulsar was not clearing up all the directories it created during installation. Whilst this is somewhat expected for the configuration directory, there were also examples of other elements being left behind. To this end, [@confused-techie] has been putting together a new package, [pulsar-cleanup](https://github.com/pulsar-edit/pulsar-cleanup), to try and deal with these leftover elements. As `electron-builder` is largely in control of the uninstall process, some elements do get left over at the end of the process. This utility will be a stand-alone executable that can be used to fully clean up all these extra artifacts, leaving your system entirely clean of any Pulsar installation.
```

TIL there's a difference in proper english and american english here.

Reply:
TIL lol. As its meant to be from the team I'll use the "English (Simplified)" version :P